.circleci/config.yml

```diff
@@ -106,27 +106,34 @@ jobs:
   publish-github:
     <<: *defaults
     docker:
-      - image: golang:1.11-alpine
+      - image: golang:alpine
     steps:
       - attach_workspace:
           at: /go/src/github.com/qlik-oss/gopherciser/
       - run:
-          name: "Publish release to github.com"
+          name: Publish to github
           command: |
-            set -e
-            apk add --no-cache git
-            go get -u github.com/tcnksm/ghr
-            TAG=v$(cat build/version)
+            set -eu
+            if [[ ! -f ./build/version ]]; then echo "./build/version not found"; exit 1; fi
+            go install github.com/github-release/github-release@latest
+            TAG=v$(cat ./build/version)
             TITLE=$(head -n 1 changelog.md)
-            BODY=$(tail -n+1 changelog.md)
-            RELEASE=pack
-            echo publish contents of $RELEASE as release $TAG
-            ghr -t ${GITHUB_TOKEN} -debug -u "${CIRCLE_PROJECT_USERNAME}" -r "${CIRCLE_PROJECT_REPONAME}" -c ${CIRCLE_SHA1} -n "$TITLE" -b "$BODY" -delete $TAG $RELEASE
+            BODY=$(tail -n+2 changelog.md)
```

Comment:
Why was it changed to n+2? Blank link?

Reply:
title got included (and those duplicated) has been an issue since always if you look at the current releases, won't include the title now.